themes/default/static/programs/aws-ec2-vpc-resources-csharp/Pulumi.yaml

```
@@ -0,0 +1,7 @@
+name: aws-ec2-vpc-resources-csharp
+runtime: dotnet
+description: An example that deploys a VPC with an internet gateway, public subnet, route table, and security group, and launches an Amazon Linux EC2 instance with NGINX.
+config:
+  pulumi:tags:
+    value:
+      pulumi:template: aws-csharp
```

Little thing, but these are cruft from having used `pulumi new`, I think; since they aren't actually necessary, we can just delete them.

```suggestion
description: An example that deploys a VPC with an internet gateway, public subnet, route table, and security group, and launches an Amazon Linux EC2 instance with Nginx.
```